tests/tracer/test_tracer.py

```diff
@@ -1213,6 +1213,13 @@ def test_partial_flush_configure_precedence(self):
         self.tracer.configure(partial_flush_enabled=True, partial_flush_min_spans=5)
         self.test_partial_flush()
 
+    @TracerTestCase.run_in_subprocess(
+        env_overrides=dict(DD_TRACER_PARTIAL_FLUSH_ENABLED="true", DD_TRACER_PARTIAL_FLUSH_MIN_SPANS="5")
+    )
+    def test_partial_flush_deprecated_config(self):
+        # Test tracer with deprecated config DD_TRACER_...
+        self.test_partial_flush()
```

Comment:
I don't think this will work how you think, since calling this method is probably running it in a subprocess with the original env overrides?

Reply:
Yes I completely missed that. I made a fix